enhancements/platform-validation.md

```diff
@@ -0,0 +1,195 @@
+---
+title: Kepler Platform Validation
+authors:
+  - Jie Ren (jie.ren@intel.com)
+reviewers:
+  - TBD
+approvers:
+  - TBD
+creation-date: 2023-05-27
+last-updated: 2023-05-27
+tracking-links: # link to related GitHub issues
+  - TBD
+---
+
+# Kepler Platform Validation Guide
+
+## Summary
+
+A quick start guide to validate Kepler support on specific hardware platform.
+
+## Motivation
+
+Currently, Kepler's test cases focus on code logic verification, lack of actual platform validation.
+
+Kepler's [data](https://github.com/sustainable-computing-io/kepler/tree/main/data) files majorly contain static information, or lack of data sync mechanism.
+
+Test cases and test report are not available, which could be the evidence or check criteria to below concerns:
+
+1. Whether the specific CPU/GPU/FPGA/Accelerator model is well supported in Kepler?
+
+2. Whether the data/metrics of specific power source are correctly exported?
+
+3. Whether the power measurement accuracy is accpetable?
+
+4. More...
+
+### Goals
+
+1. Define workflow to validate specific hardward platform support in Kepler.
+
+2. Design test cases for such validation. (Platform agnostic and specific cases)
+
+3. Define test report format and release process. (General test items result and hardware specific items)
+
+### Non-Goals
+
+1. Platforms which are not supported by Kepler yet should be out of this enhancement's scope.
+
+2. Platforms which do not support power measurement could be low priority for this enhancement, since they depends on model train and the power consumption check criteria is TBD.
+
+## Proposal
+
+We would suggest to add platform validation test cases and define the template of the test report.
+
+The specific tasks could be broken down as follows:
+
+1. Platform validation guide document.
+
+2. Platform validation test cases design and implementation.(Both platform agnostic and specific cases)
+
+3. Platform validation test report format definition, report generation and release process definition.
+
+
+### Workflow Description
+
+Step 1: Create a new local Kubernetes cluster on the test node with target hardware platform.
+
+Bring the cluster up with following [repo](https://github.com/sustainable-computing-io/local-dev-cluster) or just `make cluster-up`
+
+Step 2: Deploy Kepler in the cluster.
+
+There are three kinds of deployment available in Kepler:
+
+Deploy using Helm Chart. Follow guide [here](https://github.com/sustainable-computing-io/kepler-helm-chart/blob/main/README.md).
+
+Deploy using Kepler Operator. Follow guide [here](https://github.com/sustainable-computing-io/kepler-operator/blob/v1alpha1/README.md).
+
+Deploy from Kepler Source Code. Follow guide [here](https://sustainable-computing.io/installation/kepler/).
+
+All the above deployment scenarios related validation should be automated. i.e. locate at $Kepler_ROOT/hack/platform-validation-deploy-from-xxx.sh.
```

Comment:
as there may different mount path for k8s, for example kind vs minikube... I am good with the idea. But considering we need deploy helm and operator(basing on current document version).... I will leave an open question here, is `$Kepler_ROOT/hack/platform-validation-deploy-from-xxx.sh.` a good place to keep this script?